indigo_app/static/javascript/indigo/views/work_commencements.js

```diff
@@ -60,6 +64,24 @@
       $(e.target).closest('.card').find('.provisions-commenced').toggleClass('d-none', e.target.checked);
     },
 
+    onExpandCollapseClick: function(e) {
+      e.stopPropagation();
+      const button = e.currentTarget;
+      const parent = button.closest('li');
+      if(button.classList.contains('expanded')) {
+        button.classList.remove('expanded');
+      } else {
+        button.classList.add('expanded');
+      }
+
+      if(parent) {
```

Comment:
Surely the HTML guarantees that `parent` is not null?

You can also rely on jQuery to do all this in one go (what it's good at - it handles zero, one and many matches gracefully): 

`$(parent).find('.collapse').collapse('toggle');`

Reply:
`.find` is selects all the nodes in the parent. lets rather use
`$(parent).find('.collapse').first().collapse('toggle');`